### PR TITLE
Added About Page

### LINKED ISSUE #94: Add About page
**User story**
As a user, I want to be able to see the website description so that I can know what's the `Thinksmall` is.

**Acceptance criteria**

* Create About page that has the description of Thinksmall
* Connect with navbar menu

### PR DESCRIPTION
**Pull request by anhyesun**
#### [Added About Page](#)

**Checklist**
- [x] Run `npm run lint:fix` or `yarn lint:fix`

**What and how**
Added About page on Website

**Screenshots**
about

### CODE REVIEW

File: client/src/modules/about.module.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+.container {
+    display: flex;
+    width: 926px;
+    height: 75px;
+    margin: auto;
+}
+
+.img {
+    border-radius: 50%;
```
Comment (yehee) [inline client/src/modules/about.module.css:16]: should img be centered?

File: client/src/pages/AboutPage.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <Square 
+                title="How it works"
+                subtitle="ThinkSmall will help small business owners by providing a directory of small businesses, as well as a platform to connect small business owners who have a need for services (such as the creation of the logo, and translation for flyers) with their volunteers who have the skillset to help them."
+                src="https://images.unsplash.com/photo-1579208570378-8c970854bc23?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2295&q=80"
+            />
+        </div>
+    </div>
+)
+
+export default About
```
Comment (yehee) [inline client/src/pages/AboutPage.js]: should have extra line below

File: client/src/modules/about.module.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    height: 380px;
+    margin: 30px auto;
+    padding: 10px 10px 50px;
+    text-align: left;
+    width: 380px;
+}
+
+@media (max-width: 768px) {
+    .container {
+        display: block;
```
Comment (yehee) [inline client/src/modules/about.module.css:37]: width of the `container` class should be adjusted as well

Comment (anhyesun) [reply client/src/modules/about.module.css:37]: done!

File: client/src/modules/about.module.css (hunk 1 above)
Comment (anhyesun) [reply client/src/modules/about.module.css:16]: Tried to add the margin, but not working :( Any idea?

Comment (yehee) [inline client/src/modules/about.module.css:16]:
```suggestion
    border-radius: 50%;
    margin: auto;
    display: block;
```

Comment (anhyesun) [reply client/src/modules/about.module.css:16]: thanks!

**Review by yehee — APPROVED** [review]

**Pull request merged**